# config/perception_params.yaml
perception_node:
  ros__parameters:
    # Point cloud processing
    voxel_size: 0.005
    plane_threshold: 0.01
    plane_ransac_n: 3
    plane_num_iterations: 1000
    
    # Object clustering
    cluster_tolerance: 0.02
    min_cluster_size: 100
    max_cluster_size: 25000
    
    # Detection parameters
    min_object_height: 0.02
    max_object_height: 0.15
    workspace_limits:
      x_min: 0.2
      x_max: 0.8
      y_min: -0.4
      y_max: 0.4
      z_min: 0.0
      z_max: 0.3
    
    # Processing frequency
    processing_rate: 10.0

